src/Client.php

```diff
@@ -185,7 +185,9 @@ public function getScheduleForDay(array $channelIds, Carbon $date)
                 $scheduleItem = new ScheduleItem();
 
                 $scheduleItem->title = $item->title;
-                $scheduleItem->summary = $item->asset->summary->short;
+                if (isset($item->asset->summary)) {
```

Comment:
```suggestion
                if (isset($item->asset, $item->asset->summary, $item->asset->short)) {
```

It may be worth doing the something similar to this above, just to make sure that everything you need exists.